malaria-ios/ViewControllers/RapidFireViewController.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+class RapidFireViewController: UIViewController {
```

Comment:
Tomorrow we will talk, since you simulating retrieving the questions from the server, what would happen if it returns 0 questions/answers, etc?